cohorts/cohort.py

```diff
@@ -675,14 +702,14 @@ def _load_single_patient_merged_variants(self, patient, use_cache=True):
             else:
                 merged_variants = self._merge_variant_collections(variant_collections, self.merge_type)
         except IOError:
+            logger.warning("Error reading from variant files for patient {}".format(patient.id))
             no_variants = True
 
         # Note that this is the number of variant collections and not the number of
         # variants. 0 variants will lead to 0 neoantigens, for example, but 0 variant
         # collections will lead to NaN variants and neoantigens.
         if no_variants:
-            print("Variants did not exist for patient %s" % patient.id)
-            merged_variants = None
+            raise MissingVariantFile(patient_id=patient.id)
```

Comment:
There might be some repeated logic here; so many checks. Maybe file an issue to clean up this function? Don't want to delay the PR.